incubator/aci/create-k8-connector/script.md

```diff
@@ -61,7 +67,7 @@ spec:
     spec:
       containers:
       - name: aci-connector
-        image: microsoft/aci-connector-k8s:latest
+        image: microsoft/aci-connector-k8s:canary
```

Comment:
This is OK in incubator but we need to pin to a release version before we can graduate.

Reply:
One day - yes